article/app/model/LiveBlogHelpers.scala

```diff
@@ -112,7 +112,7 @@ object LiveBlogHelpers {
     val textBlocks = firstPageBlocks
       .take(number)
       .collect {
-        case BodyBlock(id, html, summary, title, _, _, _, publishedAt, _, updatedAt, _, _) if html.trim.nonEmpty =>
+        case BodyBlock(id, html, summary, title, _, _, _, publishedAt, _, updatedAt, _, _, _) if html.trim.nonEmpty =>
```

Comment:
rather than modifying `BodyBlock` maybe we should have a new `DotcomRenderingBodyBlock`?

Reply:
Could do! I didn't want to change too much of the underlying model because then I'll have to move the change upwards and start refactoring out the meta.java that builds up the core frontend object so I was trying to be a bit lighter touch than that, I've left it for now. Let me know what you think

Reply:
Ah yep I see what you mean - just had a look at the code as this would mean just going all the way up...it would be cleaner to have a totally separate dotcomrenderingblocks model but I agree doesn't make sense at this point - perhaps once we've worked out what bits we're using we can to the same as you did for the dotcomrendering model.